src/Bravo3/ImageManager/Services/DataInspector.php

```diff
@@ -1,23 +1,25 @@
 <?php
+
 namespace Bravo3\ImageManager\Services;
 
 use Bravo3\ImageManager\Enum\ImageFormat;
 
 /**
- * Inspects data for common formats
+ * Inspects data for common formats.
  */
 class DataInspector
 {
     /**
-     * Test to see if we can determine the image format
+     * Test to see if we can determine the image format.
      *
      * @param string $data
+     *
      * @return ImageFormat|null
      */
     public function getImageFormat(&$data)
     {
         if (!$data || strlen($data) < 5) {
-            return null;
+            return;
```

Comment:
I don't understand why, I think it's inconsistent and removes readability. Yes, please revert.